card/mod/01_core/set/all/fetch.rb

```diff
@@ -23,7 +22,7 @@ module ClassMethods
   #     :local_only                 Use only local cache for lookup and storing
   #     new: {  card opts }      Return a new card when not found
   #
-  def fetch mark, opts = {}
+  def fetch mark, opts={}
```

Comment:
Assignment Branch Condition size for fetch is too high. [22.23/15]<br>Method has too many lines. [17/15]